packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -55,8 +55,12 @@
   }
 
   .help-button {
-    margin: 15px;
+    padding: 20px;
+    position: fixed;
+    position: sticky;
+    right: 0;
     text-align: right;
+    top: 90%;
```

Comment:
this will be variable depending on the height of the screen.... why not use bottom instead?

Reply:
Right now the help-button is part of the main container. Used bottom keeping the button inside the main container. Since the sticky position is relative to the parent component the button moves to the top when there is no content like the following example.
<img width="464" alt="screen shot 2017-09-06 at 4 48 59 pm" src="https://user-images.githubusercontent.com/21693381/30134024-c23f9294-9323-11e7-85c6-fc1093323f2f.png">

Bottom works in we move the help-button outside the main container. But in that case, the help button is not relative to the content of the main container.

Reply:
help button should stay at the bottom of the page I think